tests/integration/cli/test_stake_cli_functionality.py

```diff
@@ -870,7 +871,32 @@ def test_create_interactive(click_runner,
     assert MAXIMUM_STAKE_REACHED in result.output
     assert SUCCESSFUL_STAKE_INCREASE not in result.output
 
+    # successfully stake minimum allowed which equals available balance
+    mock_staking_agent.get_locked_tokens.return_value = token_economics.minimum_allowed_locked
+    mock_token_agent.get_balance.return_value = token_economics.minimum_allowed_locked
+    min_stake_value = NU.from_nunits(token_economics.minimum_allowed_locked)
+    min_amount_user_input = '\n'.join((str(selected_index),
```

Comment:
might be helpful to include some comments here as to what each input is meant for.